### PR TITLE
docs: add seata versions maintain plan docs

### PR DESCRIPTION
**Pull request by ggbocoder**
## What is the purpose of the change

add seata versions maintain plan docs

## Brief changelog

add seata versions maintain plan docs
Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/seata/seata.github.io/issues) filed for the change(usually before you start working on it).  Trivial changes such as typos do not necessitate a GITHUB issue. Your pull request should solely focus on addressing this specific issue without incorporating any other modifications. Each PR should resolve a single issue.
- [ ] Format the pull request title like `doc: add an operations guide document`. Each commit within the pull request should contain a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `npm run build`, and make sure it works as expected.
- [ ] Ensure that no files under the `build` folder are included in the commit, as well as the `package-lock.json` and `yarn.lock` files.

### CODE REVIEW

**Review by funky-eyes — APPROVED** [review]
LGTM

File: i18n/zh-cn/docusaurus-plugin-content-docs/version-v2.0/ops/version-maintain-plan.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Apache Seata(incubating)版本维护计划
+
+------
+
+
+![version maintain plan](/img/version/version_maintain_plan.png)
+
+（上图为Apache Seata(incubating)历史版本维护的状态图）
+
+需要注意的是，社区仅维护当前时间点下最新的3个版本，并且对于非当前版本，只提供安全漏洞和bugfix相关的迭代，鼓励用户尝试最新版本的feature并给出反馈。
```
Comment (slievrly) [inline i18n/zh-cn/docusaurus-plugin-content-docs/version-v2.0/ops/version-maintain-plan.md]: How is the period calculated on the picture here, and why does 1.7 have a 6 month maintenance period and 1.8 only 3 months?

Comment (funky-eyes) [reply i18n/zh-cn/docusaurus-plugin-content-docs/version-v2.0/ops/version-maintain-plan.md]: > How is the period calculated on the picture here, and why does 1.7 have a 6 month maintenance period and 1.8 only 3 months?

This is calculated through the time node of the community release. Our maintenance time is until the release of the next official version, so the span of 1.7 is very large.

Comment (ggbocoder) [reply i18n/zh-cn/docusaurus-plugin-content-docs/version-v2.0/ops/version-maintain-plan.md]: Based on the [release dates](https://github.com/apache/incubator-seata/releases), considering that version 1.6.0 was released in December, and version 1.7.0 was released in the following July, the period between these releases is considered the stability maintenance period for version 1.6.0. With version 1.8.0 being released in October, the stability maintenance period for version 1.7.0 is between July and October.

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**